### ISSUE TITLE
Heart Failure Prediction

### ISSUE THREAD

**Issue by SaloniThete** [prathimacode-hub/PyAlgo-Tree#196]
PROJECT TITLE

Heart Failure Prediction

INTRODUCTION

Using Multiple Algorithms and comparing the accuracy for the prediction.

DATASET

[Dataset](https://drive.google.com/file/d/1xUrRvPQMs5w6RCrXESvtU4_JkTiK74U9/view?usp=sharing)

PURPOSE

predict the accuracy of heart failures using multiple algorithms.

LIBRARIES USED

pandas 
matplotlib
seaborn
from sklearn.preprocessing import LabelEncoder
from sklearn.ensemble import GradientBoostingClassifier
from sklearn.metrics import accuracy_score,confusion_matrix
from sklearn.tree import DecisionTreeClassifier
from sklearn.ensemble import RandomForestClassifier
from sklearn.neighbors import KNeighborsClassifier
from sklearn.svm import SVC
from sklearn.linear_model import LogisticRegression
from xgboost import XGBClassifier
from sklearn.naive_bayes import GaussianNB

CONCLUSION

 We get to know which algorithm is more accurate and comparison between them.

YOUR NAME

Saloni Thete

**Comment by SaloniThete** [comment]
kindly Assign me as a contributor to HACKTOBERFEST 2021.
Kindly Review & merge the PR under HACKTOBERFEST 2021
Thank you.
@prathimacode-hub

**Comment by prathimacode-hub** [comment]
Issue assigned to @SaloniThete

**Comment by SaloniThete** [comment]
kindly evaluate my PR
Pr No : #201 
Pr link : https://github.com/prathimacode-hub/PyAlgo-Tree/pull/201
@prathimacode-hub
